src/server/api/lib/campaign.ts

```diff
@@ -782,3 +793,12 @@ WHERE campaign_id = ?`,
 
   return invalidFields;
 };
+
+export const deleteCampaign = async (campaignId: string) => {
```

Comment:
Question: I believe we're still intending for deletion to only happen for template campaigns (at least for now). Should we add checking `is_template` for the campaign down here before deleting as well?